globus_sdk/exc.py

```diff
@@ -90,6 +92,7 @@ class TransferAPIError(GlobusAPIError):
     :ivar request_id: Unique identifier for the request, which should be
                       provided when contacting support@globus.org.
     """
+
```

Comment:
I know this is super-picky, but could you scrub these extra newlines and squash that with the current commit? Do that and I'll merge.

I want to avoid the noisy diff and inconsistent formatting with all of the other files.
Ultimately, I'd end up tearing back through here and fixing these lines at a later date, so it's just better to fix them now.